Documents/Management/WeeklyReports/weeklyReportFifthWeek.md

```diff
@@ -0,0 +1,36 @@
+# FifthWeeklyReport
+
+[Cumulative Weekly Reports](mainWeeklyReport.md)
+
+## Week 5 (8-10 April 2024)
+
+The fifth week was a shorter week due to the Hackathon. The team finished working with LEDs. The team also finished working on the test plan, the user manual, and the code documentation.
+
+## This week's topics
+
+1. Finish to Manage each component with LEDs.
+2. Finish to write the test plan.
+3. Finish to write the user manual.
+
+## Achievements
+
+1. Code
+   - Write properly the code with headers files.
+   - Assure the security with the Bluetooth connection.
+   - Add a function to check the battery level with percentage.
+
+2. Documents
+   - Finish the test plan on Wednesday.
+   - Finish the user manual on Wednesday.
+
+## Challenges
+
+The challenges encountered this week are still the same as the previous week but the team managed to finish the work with LEDs.
+
+## Next week's plan
+
+- Prepare the final presentation
+
+## Conclusion
+
+The fifth week was short, the team finished simulating the functionality of each component with LEDs. The team also managed to finish the documentation like the test plan and the User manual. The team is still motivated to finish the project with the presentation next week.
```

Comment:
Same as in the previous document, why there is a U in uppercase in the middle of the sentence?

Reply:
fixed